internal/cutil/aliases.go

```diff
@@ -1,12 +1,26 @@
 package cutil
 
+// #include <stdlib.h>
+// typedef void* void_ptr;
```

Comment:
I'd like to standardize on /* and */ comments for import "C" rather than //. It's no biggie but it does make things easier to expand in the future and easier to find C code via grep.

Reply:
Sure